addons/mail/static/src/xml/discuss.xml

```diff
@@ -316,6 +316,7 @@
                     <span class="o_preview_counter">
                         <t t-if="preview.unreadCounter &gt; 0">&amp;nbsp;(<t t-esc="preview.unreadCounter"/>)</t>
                     </span>
+                    <span class="o_preview_expand o_discuss_icon o_thread_window_expand fa fa-expand p-2 align-self-center" t-if="preview.isLinkedToDocumentThread"/>
```

Comment:
I would remove `p-2` (too much padding) and `align-self-center` (no visual changes).
I would instead add the following style on this element in SCSS:

```scss
.o_preview_title {
   align-items: center;
}

.o_preview_expand {
   margin: 0px 6px;
}
```